adapter.js

```diff
@@ -1,207 +1,82 @@
-////////////////////////////////////////////////////////////////////////////
-//
-// Copyright 2016 Realm Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-////////////////////////////////////////////////////////////////////////////
-
-'use strict';
-
+const PostgresAdapter = require('realm-data-adapters').PostgresAdapter
+const Realm = require('realm')
 const fs = require('fs');
 const path = require('path');
-const Realm = require('realm');
-const PostgresAdapter = require('./postgres_adapter.js');
-const PostgresClient = require('libpq');
-
-// CONFIGURABLE
-var access_token = 'ewoJImlkZW50aXR5IjogIl9fYXV0aCIsCgkiYWNjZXNzIjogWyJ1cGxvYWQiLCAiZG93bmxvYWQiLCAibWFuYWdlIl0KfQo=:UH4womynqiP+9/b4kXvGc7E9XrPB3bbdCKJNXhjbWwXN7jbRLlldsG5jCu5CF9vEIYl8nSrORmGpIC/6tztlPzKs3RkIphJWb19uvD3shL6zKy7W+iZrGzPrFDNpEXM4hp4Hq1L/zgxMf1vlElL5JzW2Yq5AybpZZAeP6IVM/xC5bRfCzf7iER8om28WxnVK3CC2buOJbV+dKKWNSVbVl7Ol3N9+Vu2owkByGGGaY0DjzWkTzf+Y+q8r1aoC5MBgOqXhde+bWSiKc2hzFK5FZH1m0KG5+odD40r2mahxdAw304S6ipJcIwGA5pFlBsSNQ/c9K8DYvR0sUZXu9M5xlQ==';
```

Comment:
Running on your system is not the same as being able to work with any given system. This code is definitely necessary given that it removes a foreign key constraint in the DVDRental database so it will definitely fail without that.